src/Common/TestCodeRules.ruleset

```diff
@@ -4,6 +4,8 @@
   <Rules AnalyzerId="Microsoft.Analyzers.ManagedCodeAnalysis" RuleNamespace="Microsoft.Rules.Managed">
     <Rule Id="CA1014" Action="None" />
     <Rule Id="CA1020" Action="None" />
+    <Rule Id="CA1062" Action="None" />
```

Comment:
These are rules for the tests (acceptance tests, unit tests). I am not very keen to do it there.